### PR TITLE
Lenovo Miix3-830 sensor matrix to hwid + doc update

### PR DESCRIPTION
**Pull request by k3a**
Added Lenovo Miix 3-830 hwid to correct accelerometer orientation. 
Tested on the actual HW.

Improved doc in the comment on how to get the proper sensor match string

### CODE REVIEW

**Comment by k3a** [comment]
Also here is the export of udev db as requested.
[miix-udev-db.txt](https://github.com/systemd/systemd/files/2219183/miix-udev-db.txt)

**Comment by hadess** [comment]
This needs to be split into 2 separate commits. Please also try to follow the style used by commit messages to this same file.

**Comment by k3a** [comment]
Updated as requested

File: hwdb/60-sensor.hwdb [hunk 1]
```diff
@@ -11,6 +11,16 @@
 # Match string formats:
 # sensor:modalias:<parent device modalias>:dmi:<dmi string>
 #
+# The device modalias can be seen in the `modalias` file 
+# of the sensor parent, for example:
+# cat /sys/`udevadm info -q path -n /dev/iio:device0`/../modalias
+#
+# The full "dmi string" of the running machine can be read from
```
Comment (hadess) [inline hwdb/60-sensor.hwdb]: "DMI" not "dmi"

File: hwdb/60-sensor.hwdb [hunk 2]
```diff
@@ -11,6 +11,16 @@
...
+# /sys/class/dmi/id/modalias
+# That requires a kernel built with CONFIG_DMIID set, which is common.
+# The full dmi string is not needed here and the meaning of individual parts
```
Comment (hadess) [inline hwdb/60-sensor.hwdb]: Same here.

File: hwdb/60-sensor.hwdb [hunk 3]
```diff
@@ -11,6 +11,16 @@
...
+# can be seen in the source of the DMIID kernel module.
```
Comment (hadess) [inline hwdb/60-sensor.hwdb]: Link to the upstream sources here?

File: hwdb/60-sensor.hwdb [hunk 4]
```diff
@@ -11,6 +11,16 @@
...
+#
 # To add local entries, create a new file
 #   /etc/udev/hwdb.d/61-sensor-local.hwdb
 # and add your rules there. To load the new rules execute (as root):
```
Comment (hadess) [inline hwdb/60-sensor.hwdb]: Please also cut down the commit subject. Something like:
```
hwdb: Add information on where to find identifiers

Update sensor hwdb comments to clarify where the modalias and DMI
string can be gathered from.
```

**Comment by k3a** [comment]
Nits removed + link to the upstream source added

File: hwdb/60-sensor.hwdb [hunk 5]
```diff
@@ -11,6 +11,17 @@
...
+# The device modalias can be seen in the `modalias` file 
+# of the sensor parent, for example:
+# cat /sys/`udevadm info -q path -n /dev/iio:device0`/../modalias
+#
+# The full DMI string of the running machine can be read from
+#   /sys/class/dmi/id/modalias
+# That requires a kernel built with CONFIG_DMIID set, which is common.
+# The full DMI string is not needed here and the meaning of individual parts
+# can be seen in the source of the DMIID kernel module
+#   https://github.com/torvalds/linux/blob/master/drivers/firmware/dmi-id.c
```
Comment (hadess) [inline hwdb/60-sensor.hwdb]: It's actually here:
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/drivers/firmware/dmi-id.c

**Comment by hadess** [comment]
Looks good to commit after fixing that link. Thanks!

**Comment by k3a** [comment]
A good exercise on git rebase :) Thanks for maintaining the project.

**Comment by hadess** [comment]
> A good exercise on git rebase :) Thanks for maintaining the project.

Sorry about that :)

**Pull request merged**